### ISSUE TITLE
mirror repo doesn't contain InRelease and RepoVer from original repo.

### ISSUE THREAD

**Issue by asukahan** [apt-mirror/apt-mirror#159]
i mirrored:
http://download.hiveos.farm/repo/binary/

with

deb http://download.hiveos.farm/repo/binary/ /

in mirror.list file.

everything goes fine and downloaded correctly but InRelease and RepoVer files from original repo. is this an issue or I need to modify my configuration? thx. 

wget those 2 files, worked.

**Comment by asukahan** [comment]
############# config ##################
#
# set base_path    /var/spool/apt-mirror
#
# set mirror_path  $base_path/mirror
# set skel_path    $base_path/skel
# set var_path     $base_path/var
# set cleanscript $var_path/clean.sh
# set defaultarch  <running host architecture>
# set postmirror_script $var_path/postmirror.sh
# set run_postmirror 0
set nthreads     20
set _tilde 0
#
############# end config ##############

#deb http://archive.ubuntu.com/ubuntu artful main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-security main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-updates main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-proposed main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-backports main restricted universe multiverse

#deb-src http://archive.ubuntu.com/ubuntu artful main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-security main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-updates main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-proposed main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-backports main restricted universe multiverse

#clean http://archive.ubuntu.com/ubuntu

deb http://download.hiveos.farm/repo/binary /

**Comment by asukahan** [comment]
```

############# config ##################
#
# set base_path    /var/spool/apt-mirror
#
# set mirror_path  $base_path/mirror
# set skel_path    $base_path/skel
# set var_path     $base_path/var
# set cleanscript $var_path/clean.sh
# set defaultarch  <running host architecture>
# set postmirror_script $var_path/postmirror.sh
# set run_postmirror 0
set nthreads     20
set _tilde 0
#
############# end config ##############

#deb http://archive.ubuntu.com/ubuntu artful main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-security main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-updates main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-proposed main restricted universe multiverse
#deb http://archive.ubuntu.com/ubuntu artful-backports main restricted universe multiverse

#deb-src http://archive.ubuntu.com/ubuntu artful main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-security main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-updates main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-proposed main restricted universe multiverse
#deb-src http://archive.ubuntu.com/ubuntu artful-backports main restricted universe multiverse

#clean http://archive.ubuntu.com/ubuntu

deb http://download.hiveos.farm/repo/binary /

```

**Comment by asukahan** [comment]
```
~# /usr/bin/apt-mirror
Downloading 5 index files using 5 threads...
Begin time: Tue Jul  5 00:39:21 2022
[5]... [4]... [3]... [2]... [1]... [0]...
End time: Tue Jul  5 00:39:21 2022

Processing translation indexes: [T]

Downloading 0 translation files using 0 threads...
Begin time: Tue Jul  5 00:39:21 2022
[0]...
End time: Tue Jul  5 00:39:21 2022

Processing DEP-11 indexes: [D]

Downloading 0 dep11 files using 0 threads...
Begin time: Tue Jul  5 00:39:21 2022
[0]...
End time: Tue Jul  5 00:39:21 2022

Processing indexes: [P]

9.6 GiB will be downloaded into archive.
Downloading 1570 archive files using 20 threads...
Begin time: Tue Jul  5 00:39:21 2022
[20]... [19]... [18]... [17]... [16]... [15]... [14]... [13]... [12]... [11]... [10]... [9]... [8]... [7]... [6]... [5]... [4]... [3]... [2]... [1]... [0]...
End time: Tue Jul  5 00:52:28 2022

0 bytes in 0 files and 0 directories can be freed.
Run /var/spool/apt-mirror/var/clean.sh for this purpose.

Running the Post Mirror script ...
(/var/spool/apt-mirror/var/postmirror.sh)

```

**Comment by asukahan** [comment]
```
~# wget http://download.hiveos.farm/repo/binary/RepoVer
--2022-07-05 01:37:35--  http://download.hiveos.farm/repo/binary/RepoVer
Resolving download.hiveos.farm (download.hiveos.farm)... 104.22.11.47, 172.67.28.84, 104.22.10.47, ...
Connecting to download.hiveos.farm (download.hiveos.farm)|104.22.11.47|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 8 [application/octet-stream]
Saving to: ‘RepoVer’

RepoVer                            100%[==============================================================>]       8  --.-KB/s    in 0s

2022-07-05 01:37:35 (786 KB/s) - ‘RepoVer’ saved [8/8]

```

**Comment by chutzimir** [comment]
There is indeed some problems with the flat repository format support which are probably causing issues with the InRelease. I am looking into those for another issue.

I don't know about RepoVer, is it documented somewhere and is it required by the tools to work with repositories? For custom files like this you could use a post script that apt-mirror can use to get the file for you.

**Comment by asukahan** [comment]
> There is indeed some problems with the flat repository format support which are probably causing issues with the InRelease. I am looking into those for another issue.
> 
> I don't know about RepoVer, is it documented somewhere and is it required by the tools to work with repositories? For custom files like this you could use a post script that apt-mirror can use to get the file for you.

thx a lot

**Comment by chutzimir** [comment]
@asukahan This should be fixed in #156 
You can try it out
